CHANGELOG_PENDING.md

```diff
@@ -1,5 +1,6 @@
 ### SDK Features
-
+* `service/cloudfrontkeyvaluestore`: Deprecate cloudfrontkeyvaluestore
+  * This change removes the cloudfrontkeyvaluestore service, since it does not support sigv4a.
```

Comment:
We should put the blank line back, given the known strictness of the thing that parses this.

Reply:
yah, i agree. i would try to make it look as close as possible to previous changelogs that succeeded